src/rpc/core_rpc_server.cpp

```diff
@@ -3312,6 +3313,7 @@ namespace cryptonote { namespace rpc {
   //------------------------------------------------------------------------------------------------------------------------------
   LOKINET_PING::response core_rpc_server::invoke(LOKINET_PING::request&& req, rpc_context context)
   {
+    std::copy(req.version.begin(),req.version.end(),m_core.lokinet_version.begin());
```

Comment:
Was this debugging that should be taken out?  It seems like this is replacing the version that lokinet is trying to tell us?

Reply:
Oh, scratch that, I read it backwards.

Reply:
I *think* this can be simplified to `m_core.lokinet_version = req.version;`